### DISCUSSION TITLE
how to run this app from a sub-directory?

### DISCUSSION THREAD

**Discussion by waldenn** [iptv-org/community#534] [Q&A]
Hi, I would like to know how to run this app from a sub-directory (for my https://conze.pt project).

I can build and run the app fine from the "/" space, but I'm struggling to get it to run properly from a subdirectory.

I tried some things: 
  *  "vite build --base=/my/public/path/"
  *  and editing some source file paths

...but have not been able to get it working. Can someone help me out with this? 

Thanks for this wonderful IPTV project!

**Comment by freearhey** [comment]
If you run the command `npm run build`, the `/docs` folder will be created in the root directory containing all the site files. You only need to copy it to your project and rename it if necessary.
